src/style.css

```diff
@@ -0,0 +1,118 @@
+.page__content {
+  margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+  margin-left: 19px;
+}
+
+.card__description {
+  box-sizing: border-box;
```

Comment:
move it to `*` selector (should be applied to all elements)

Reply:
'+'